resources/views/errors/401.blade.php

```diff
@@ -0,0 +1,8 @@
+@extends('errors::minimal')
+
+@section('code', '401')
+@section('message')
+    Access Denied! This zone is exclusive to authorized attendees.<br>
```

Comment:
Minor nitpick but change it to:
```suggestion
    Access Denied! This page is exclusive to authorized attendees.<br>
```